### PR TITLE
Travis ci

### PR DESCRIPTION
**Pull request by wesleybalvanz-wf**
This adds a .travis.yml and removes the smithy.yml from the repo now that it's public also added pypi server upload to the .travis.yml so on release cuts it will be uploaded to the public pypi server.

@jacobmoss-wf @matthinrichsen-wf @ericolson-wf @rosshendrickson-wf  @alexandercampbell-wf

### CODE REVIEW

**Comment by jacobmoss-wf** [comment]
+1

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+script:
+- flake8
+- nosetests
+after_script:
+- codecov -u https://codecov.io --no-fail
+deploy:
+  provider: pypi
+  user: wesleybalvanz-wf
+  password:
+    secure: sFB1+BIdh4KDs+PuR+wAxOI0abVse1YEzSg04UZ/jGkuJeAxo2pCLFFzo5cwWEv6T9mVN1lVoC4ZnLfD0mkasXoq3mdATHX9uik8F4B7dQR+Azyl/uS1y/3eAxCOUkC6dSQj3USoWXJWf6dDbRYWcMgMhSUSylx3Dg2GH1xYlrEpdzydNXWIICTNTFURtO7QmYXlVSncjJhSWaKmkTikulgtcklwIp1GrLd7NkQeLMHpeksOLRgJLDA5PV/FopYNV4aeREkyDaGwWul73zwTWp6uriX6G2+Kk+JLco5IP5Jm60t5Xi/c7dBiVBb2JgEr5lXPyZWu4NBdNsJNXcOTc6mabjzBGr4O2u0IvTN21VpEfaW2TVQqY6qpDaCmqVjsWewdZAHuNzfddeNcxRy/JWOaD+2PfVXnCnfwt7QvM+LAkPE4rmSF/j8yA9Meyfyg9PWiFZVzhKIe/gGd6Pio8/EkLJLL5GuQDJcGtE+nuekNpGyKoK15LcVqhQBU5WLivPPLJ3U5e4RE7A/k23xhjF4V0r8Il6zWDdRazvYvsJL0KnvcjQY6pFW0bTi7H3KSzcTCIET9+d3RhqVKVb8S/6UEiUegC6fLowtL667PO6Dq2vjGFVMn4jcM9PfVQoT6HDn3aF0+ge7rG4gFKbyoWTGA/pma1WqkD3US2rRVNIc=
```
Comment (alexandercampbell-wf) [inline .travis.yml]: Is this your `wesleybalvanz-wf` password for pypi?

Comment (wesleybalvanz-wf) [inline .travis.yml]: I was just following the travis.ci docs on how to add pypi: https://docs.travis-ci.com/user/deployment/pypi I do see how this in some ways isn't ideal yes.

Comment (alexandercampbell-wf) [inline .travis.yml]: Looks like it's encrypted. Since it's advised in the official Travis docs, I suppose it must be secure.

**Comment by alexandercampbell-wf** [comment]
+1

File: .travis.yml (hunk 1 above)
Comment (matthinrichsen-wf) [inline .travis.yml]: "I know - lets have them copy-paste their password and make it publicly available." - TravisCI
...did these guys happen to previously work at Sony?

**Pull request merged**